meta/main.yml

```
@@ -1,11 +1,12 @@
 ---
 dependencies: []
 galaxy_info:
-  author: First Last
+  author: Alexander King
   company: CISA Cyber Assessments
-  description: Skeleton Ansible role
+  description: This role installs the Admiral Python Library for Certificate Transparency log scanning.
   galaxy_tags:
-    - skeleton
+    - python
```

Add a couple other relevant tags:
```suggestion
    - certificate
    - certificatetransparency
    - python
```

---

Did this commit get overwritten?

---

Oops! Yes, my bad. I forgot to pull down your suggestions before force pushing.